webui/src/Page/Survey.elm

```diff
@@ -1477,8 +1477,8 @@ viewIpsativeSurveyBoxes surveyQuestion =
 
 viewSurveyBox : IpsativeAnswer -> Html Msg
 viewSurveyBox answer =
-    div [ class "col-md-6 pt-5" ]
-        [ div [ class "question-container", style "height" "170px" ]
```

Comment:
What I was crudely attempting here was to prevent the position of the sliders from shifting around on the screen as the user transitioned from one question to the next on desktop. Because the amount of text in the questions varies, when transitioning from one question to the next the sliders move around - it's more apparent on some questions than others. I was attempting to trade off having extra space inside the boxes for the questions with less text in the answers in exchange for having the sliders stay in the same spot from screen to screen. Totally open to better ways of solving this,I definitely don't want to mess up responsive functionality.

Reply:
Gotcha, I'll take a look at that tomorrow and see if there's something that can be done about that.